api/crl.go

```diff
@@ -0,0 +1,24 @@
+package api
+
+import (
+	"encoding/pem"
+	"net/http"
+)
+
+// CRL is an HTTP handler that returns the current CRL in PEM format
+func (h *caHandler) CRL(w http.ResponseWriter, r *http.Request) {
+	crlBytes, err := h.Authority.GenerateCertificateRevocationList(false)
+
+	if err != nil {
+		w.WriteHeader(500)
+		return
+	}
+
+	pemBytes := pem.EncodeToMemory(&pem.Block{
+		Type:  "X509 CRL",
+		Bytes: crlBytes,
+	})
```

Comment:
We should use the DER format. At least by default/

Reply:
I added a http param to select the format, eg:

`https://ca.example.com/crl` returns `DER`
`https://ca.example.com/crl?pem` returns `PEM`